src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -243,14 +243,13 @@ public void DirectoryLongerThanMaxLongPath_ThrowsPathTooLongException()
 
         [ConditionalFact(nameof(LongPathsAreNotBlocked), nameof(UsingNewNormalization))]
         [ActiveIssue(20117, TargetFrameworkMonikers.Uap)]
-        [PlatformSpecific(TestPlatforms.Windows)]  // long directory path with extended syntax throws PathTooLongException
-        public void DirectoryLongerThanMaxLongPathWithExtendedSyntax_ThrowsPathTooLongException()
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void DirectoryLongerThanMaxLongPathWithExtendedSyntax_ThrowsException()
         {
             var paths = IOInputs.GetPathsLongerThanMaxLongPath(GetTestFilePath(), useExtendedSyntax: true);
-            Assert.All(paths, (path) =>
-            {
-                Assert.Throws<PathTooLongException>(() => Create(path));
-            });
+
+            // Long directory path with extended syntax throws PathTooLongException.
+            Assert.All(paths, path => { Assert.Throws<PathTooLongException>(() => Create(path)); });
```

Comment:
OK, that makes a lot more sense. I have adjusted that test case accordingly. Hoping it passes everywhere now. 🤷‍♂️